akka-actor-typed/src/main/scala/akka/actor/typed/ActorRef.scala

```diff
@@ -42,6 +42,10 @@ trait ActorRef[-T] extends RecipientRef[T] with java.lang.Comparable[ActorRef[_]
    */
   def unsafeUpcast[U >: T @uncheckedVariance]: ActorRef[U]
 
+  def unsafeCast[U](clazz: Class[U]): ActorRef[U]
```

Comment:
Not sure should it be 
`def unsafeCast[U >: T @uncheckedVariance]: ActorRef[U]` or not. because it's more dangerous.